code/src/components/TaskList.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import tasks from 'reducers/tasks'
+import styled from 'styled-components'
```

Comment:
A tip is to write 'styled-components/macro' then in the Dev-tools you can easier see where the style is instead of only the random numbers and letters